101-fast-bonsai/azuredeploy.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "type": "string",
+      "defaultValue": "my-first-bonsai",
+      "metadata": {
+        "description": "The name of the Bonsai suite instance."
+      }
+    },
+    "location": {
+      "type": "string",
+      "allowedValues": [
+        "West US",
```

Comment:
is this supported in USGov?  If not, we need to add the environments property to metadata.json... if so, need to update the allowedValues here